docs/usage.md

```diff
@@ -163,22 +163,17 @@ by Jest.
 
 ## Using multiple presets
 
-TODO
-All Neutrino commands support the `--use` command line parameter, but having to specify this for each script target
-can be cumbersome and verbose, especially if you have many middleware or presets. Fortunately, Neutrino also supports
-specifying presets using the `use` property in a `.neutrinorc.js` file. By omitting the `--use`
-flag and specifying a `use` array, every call to a Neutrino command will look up which middleware
+Neutrino supports loading multiple presets and middleware using the `use` property in a
+`.neutrinorc.js` file. By specifying a `use` array, every Neutrino output function will look up which middleware
 are configured in your in `.neutrinorc.js`.
 
-This is the recommended approach when using more than one preset/middleware.
-
 ```js
 // package.json
 {
   "scripts": {
-    "start": "neutrino start",
-    "build": "neutrino build",
-    "test": "neutrino test"
+    "start": "webpack-dev-server --mode development",
+    "build": "webpack --mode production",
+    "test": "jest"
```

Comment:
Should we add a `lint` entry here and say `@neutrinojs/airbnb` to the preset lists in this multi-preset example? (In lieu of a linting section on this page; or even in addition to, should we add one?)

Reply:
It probably wouldn't hurt.